src/scripts/main.js

```diff
@@ -1,3 +1,180 @@
 'use strict';
 
-// write code here
+const theadList = document.querySelector('thead');
+const tbody = document.querySelector('tbody');
+
+theadList.addEventListener('click', (e) => {
+  const index = e.target.cellIndex;
+  const element = e.target.closest('th');
+  const tbodyRows = [...tbody.querySelectorAll('tr')];
+
+  if (!element.hasAttribute('direction')
+    || element.getAttribute('direction') === 'DESC') {
+    element.setAttribute('direction', 'ASC');
+  } else {
+    element.setAttribute('direction', 'DESC');
+  }
```

Comment:
I'd recommend:
1) move condition to a variable
2) apply ternary for this condition